features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/event/TimelineItemContentView.kt

```diff
@@ -95,6 +95,7 @@ fun TimelineItemEventContentView(
         )
         is TimelineItemPollContent -> TimelineItemPollView(
             content = content,
+            state = content.presenter.present(content),
```

Comment:
Neat: maybe create a fun in `TimelineItemPollContent` to avoid having receiver and parameter to be the same here:

```kotlin
    @Composable
    fun present(): PollContentState {
        return presenter.present(content = this)
    }
```

so this line would become:

```kotlin
state = content.present(),
```